File: specification/specs/x_pack/roll_up/stop_rollup_job/StopRollupJobRequest.ts
@rest_spec_name("rollup.stop_job")
class StopRollupJobRequest extends RequestBase {
  path_parts?: {
    id: string;
  }
  query_parameters?: {
    timeout?: Time;
    wait_for_completion?: boolean;
  }
  body?: {
  }
}

File: output/csharp/XPack/RollUp/StopRollupJob/StopRollupJobRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class StopRollupJobRequest : RequestBase {
		
		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }


		[DataMember(Name="wait_for_completion")]
		public bool WaitForCompletion { get; set; }

	}
}
